portal/add_edit_event_user.php

```diff
@@ -40,6 +40,8 @@
 require_once("$srcdir/patient.inc");
 require_once("$srcdir/forms.inc");
 
+use OpenEMR\Core\Header;
```

Comment:
Is this being used below?

Reply:
Prob left because changed mind in this case.